.github/workflows/test.yml

```diff
@@ -50,11 +50,18 @@ jobs:
             include: "sale_procurement_group_by_commitment_date"
             name: test with OCB
           - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
-            exclude: "sale_order_line_sequence,sale_procurement_group_by_commitment_date"
+            include: "sale_stock_line_customer_ref"
```

Comment:
You mustn't touch this file manually, but through copier.

Reply:
I didn't know, thanks for pointing it out.

Reply:
to be rolled back if possible, see comments below